pkg/policyconditions/conditions.go

```diff
@@ -86,7 +86,7 @@ func SetPolicyFailedToConfigure(conditions *nmstate.ConditionList, message strin
 	)
 }
 
-func nodesRunningNmstate(cli client.Client) ([]corev1.Node, error) {
+func NodesRunningNmstate(cli client.Client) ([]corev1.Node, error) {
```

Comment:
Can we do some boy scout here and mov this to `pkg/node` with something like `node.FindNmstateRunning` since  this has nothing to do with policyconditions

Reply:
Done